client/my-sites/promote-post-i2/components/payment-links/style.scss

```diff
@@ -0,0 +1,52 @@
+.promoted-posts__payment-links-container {
+	margin-left: auto;
+	margin-right: auto;
+	max-width: 1040px;
+	width: calc(100% - 128px);
+	background: var(--studio-gray-0);
+	border-radius: 4px;
+	margin-bottom: 24px;
+
+	@media screen and (max-width: 1069px) {
+		width: calc(100% - 32px);
+	}
+
+	.promoted-posts__payment-links {
+		display: flex;
+		flex-direction: column;
+		gap: 8px;
+		padding: 16px;
+
+		.promoted-posts__payment-link-row {
+			display: grid;
+			grid-template-columns: 40% 40% 20%;
+
+			.payment-link__label {
+				font-size: 0.875rem;
+				font-weight: 500;
+			}
+
+			.payment-link__link {
+				justify-self: flex-end;
+				a {
+					display: flex;
+					align-items: center;
```

Comment:
It seems that the current value produces an external link icon to be located at the bottom, like this:

<img width="281" alt="image" src="https://github.com/user-attachments/assets/c7e08141-ffc5-4a63-9e75-6ded33248136" />

I assume that using `align-items: flex-start;` instead will work better:

<img width="275" alt="image" src="https://github.com/user-attachments/assets/d3719d3c-2391-44fb-9de0-dcae464ff68f" />

What do you think?